packages/frontity-org-theme/src/components/scroll-button.tsx

```diff
@@ -0,0 +1,42 @@
+import { connect, styled } from "frontity";
+import { Connect } from "frontity/types";
+import React from "react";
+
+import FrontityOrg from "../../types";
+
+const ScrollButton: React.FC<Connect<FrontityOrg, any>> = ({ state }) => {
+  const scrollToTop = () => {
+    window.scrollTo({
+      top: 0,
+      behavior: "smooth",
+    });
+  };
+
+  const Button = styled.div`
```

Comment:
I now see that you did this because you need some data from the state, right? I think it best to style the button with the `css` prop. The component will look like this:

```js
	import { css } from "styled-components";

	const ScrollButton: React.FC<Connect<FrontityOrg>> = ({ state }) => {
		....

		return (
			<Button css={ css`
				background-color: ${state.theme.colors.frontity};
				color: ${state.theme.colors.white};

				/* other styles */
			` } onClick={ scrollToTop }> ↑ </Button>
		);
	}

	export default connect(ScrollButton);
```

Reply:
You're right, I included it inside the component because I needed to access the `state`. I've moved the properties where I needed the `state` to the `css` property as you suggested.